src/views/ProjectEdit/Framework/FrameworkDetail/index.tsx

```diff
@@ -0,0 +1,315 @@
+                                )}
+                            >
+                                {_ts('projectEdit', 'selectedFrameworkTagLabel')}
+                            </Tag>
+                        )}
+                        <QuickActionLink
+                            variant="inverted"
+                            title={_ts('projectEdit', 'editFrameworkLinkTitle')}
+                            disabled={disableAllButtons}
+                            to={reverseRoute(
+                                pathNames.analyticalFramework,
+                                {
+                                    analyticalFrameworkId: frameworkId,
+                                },
+                            )}
+                        >
+                            <Icon
+                                name="edit"
+                            />
+                        </QuickActionLink>
+                        <QuickActionButton
+                            title={_ts('projectEdit', 'cloneFrameworkButtonTitle')}
+                            variant="inverted"
+                            disabled={disableAllButtons}
+                            onClick={handleFrameworkCloneClick}
+                            name="clone"
+                        >
+                            <IoCopyOutline />
+                        </QuickActionButton>
+                    </>
+                )}
+                contentClassName={styles.content}
+            >
+                <div className={styles.leftContainer}>
+                    <div className={styles.frameworkDescription}>
+                        {frameworkDetails?.description}
+                    </div>
+                    <TextOutput
+                        className={styles.block}
+                        label={_ts('projectEdit', 'frameworkCreatorTitle')}
+                        value={frameworkDetails?.createdByName}
+                        type="small-block"
+                    />
+                    <TextOutput
+                        className={styles.block}
+                        label={_ts('projectEdit', 'analyticalFrameworkTitle')}
+                        value={(
+                            <Button
+                                variant="action"
+                                name="openImageButton"
+                                onClick={showReferenceImage}
+                            >
+                                {_ts('projectEdit', 'referenceFrameworkImageLabel')}
+                            </Button>
+                        )}
+                        type="small-block"
+                    />
+                    <TextOutput
+                        className={styles.block}
+                        label={_ts('projectEdit', 'recentlyUsedInProjectsTitle')}
+                        type="small-block"
+                        value={(
+                            <List
+                                data={frameworkDetails?.visibleProjects ?? []}
```

Comment:
```suggestion
                                data={frameworkDetails?.visibleProjects}
```
List should handle this!

Reply:
it does not

Reply:
It should!